extensions/resource-deployment/src/ui/model.ts

```diff
@@ -41,4 +60,15 @@ export class Model {
 			process.env[propertyName] = value === undefined ? '' : value;
 		});
 	}
+
+	public interpolateVariableValues(inputValue: string): string {
```

Comment:
These functions could really use doc comments explaining what they're doing. And can you come up with a better name? "interpolate" is way too generic